asciidoc/runtime/src/main/java/io/quarkiverse/qute/web/asciidoc/runtime/AsciidocConverter.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkiverse.qute.web.asciidoc.runtime;
+
+import java.util.Map;
+
+import io.yupiik.asciidoc.model.Document;
+import io.yupiik.asciidoc.parser.Parser;
+import io.yupiik.asciidoc.renderer.html.AsciidoctorLikeHtmlRenderer;
+
+public class AsciidocConverter {
+
+    private final Parser parser = new Parser();
+
+    public String apply(String asciidoc) {
+        Document document = parser.parse(asciidoc, new Parser.ParserContext(null));
+        AsciidoctorLikeHtmlRenderer renderer = new AsciidoctorLikeHtmlRenderer(new AsciidoctorLikeHtmlRenderer.Configuration()
```

Comment:
The renderer could be re-used for multiple files right,

Reply:
Yes, I will change it.

Reply:
> The renderer could be re-used for multiple files right?

In that case, both `Parser` and `AsciidoctorLikeHtmlRenderer` should be thread-safe. Is that so?